### ISSUE TITLE
Custom hide all fees below level option

### ISSUE THREAD

**Issue by dhimmel** [jhoenicke/mempool#17]
For a long time, I was unaware of the feature described as:

> You can click on some fee level in the legend to hide all fee levels below that level. This way you can better see how many transactions are competing with that fee level.

I'm glad I read the documentation, this feature is incredible for ignoring the irrelevant portion of the plot. Specifically, when deciding what fee to pay, only higher (not lower) paying transactions matter. In the screenshot below, I'm ignoring all fees below 200 sat/B:

![mempool](https://user-images.githubusercontent.com/1117703/33746041-7fd0283a-db88-11e7-9e7c-e228d7c47cd6.png)

However, the fee levels are still too wide to enable super-optimized fee determination. For example, what's the mempool size over time at 270 sat/B? It would be helpful to be able to set a custom "ignore below" fee level. In the above example, if one could manually evaluate the mempool size for any fee level between 200 and 300, they could more fully optimize their transaction fee.

I'm not sure about the technical feasibility or the best design. @jhoenicke what do you think?

On an unrelated note, I think the "Mempool Size in MB" plot should be moved to the top, since it's most relevant to the most users. Block size is the actual limited resource (not transaction count or pending fees) that users are competing for when determining what fee to pay. And the effect of mempool clearing causing by a single block is most predictable with this plot.

**Comment by Giszmo** [comment]
I agree with the new sorting of the charts. Transaction count is getting less and less relevant and size and value are certainly more relevant for almost all users.
